src/styles/controls.less

```diff
@@ -0,0 +1,537 @@
+@import './base/animations.less';
+@import './base/helpers.less';
+@import './base/mixins.less';
+@import './base/reset.less';
+@import './base/settings.less';
+@import './base/structure.less';
+@import './base/typography.less';
+
+// Player Controls
+
+//Vars
+
+@size-player-controls-button-height: 15px;
+@size-player-controls-button-height-mini: 15px;
+
+//
+
+.media-control[data-media-control]
+.media-control-layer[data-controls] {
+  display: flex;
+  position: static;
+
+  button {
+    &:extend(.hover-opacity all);
+    background-repeat: no-repeat!important;
+    background-position: center center;
+    background-color: @color-grey!important;
+    mask-repeat: no-repeat;
+    mask-position: center;
+    -webkit-mask-repeat: no-repeat;
+    -webkit-mask-position: center;
+
+    &:hover {
+      background-color: @color-white!important;
+    }
+
+    .query-mini({
+      height: @size-player-controls-button-height-mini;
+    });
+
+    svg {
+      display: none;
```

Comment:
By default, `Clappr` inserts their own `svgs` to the buttons via `javascript`. To override those and use our own (via `css`), I am simply hiding all of the inline `svg`s.